pkg/daemon/file_writers.go

```diff
@@ -276,10 +276,11 @@ func lookupGID(group string) (int, error) {
 // This is essentially ResolveNodeUidAndGid() from Ignition; XXX should dedupe
 func getFileOwnership(file ign3types.File) (int, int, error) {
 	uid, gid := 0, 0 // default to root
+	var err error    // create default error var
 	if file.User.ID != nil {
 		uid = *file.User.ID
 	} else if file.User.Name != nil && *file.User.Name != "" {
-		uid, err := lookupUID(*file.User.Name)
+		uid, err = lookupUID(*file.User.Name)
```

Comment:
Oh that is a... highly unfortunate error

Reply:
yeah, I was so confused where it was getting lost! Very easy to miss :')